src/aspire/operators/polar_ft.py

```diff
@@ -2,50 +2,60 @@
 
 import numpy as np
 
-from aspire.basis import Basis
-from aspire.nufft import anufft, nufft
+from aspire.image import Image
+from aspire.nufft import nufft
 from aspire.utils import complex_type
 
 logger = logging.getLogger(__name__)
 
 
-class PolarBasis2D(Basis):
+class PolarFT:
     """
     Define a derived class for polar Fourier representation for 2D images
     """
 
     def __init__(self, size, nrad=None, ntheta=None, dtype=np.float32):
         """
-        Initialize an object for the 2D polar Fourier grid class
+        Initialize an object for the 2D polar Fourier grid class. `PolarFT` expects that
+        images are real and uses only half of the `ntheta` values. Downstream algorithms should
+        take advantage of the conjugate symmetry of the polar Fourier coefficients if the full
+        set is needed.
 
         :param size: The shape of the vectors for which to define the grid.
             May be a 2-tuple or an integer, in which case a square basis is assumed.
             Currently only square images are supported.
         :param nrad: The number of points in the radial dimension. Default is resolution // 2.
         :param ntheta: The number of points in the angular dimension. Default is 8 * nrad.
+        :param dtype: dtype of polar Fourier grid.
         """
         if isinstance(size, int):
             size = (size, size)
         ndim = len(size)
         assert ndim == 2, "Only two-dimensional grids are supported."
         assert len(set(size)) == 1, "Only square domains are supported."
 
+        self.ndim = ndim
+        self.sz = size
         self.nrad = nrad
         self.ntheta = ntheta
+        self.dtype = dtype
 
-        super().__init__(size, dtype=dtype)
+        self._build()
 
         # this basis has complex coefficients
         self.coefficient_dtype = complex_type(self.dtype)
 
+        # Store the "half" transform for use in building the full transform.
+        self._transform = None
```

Comment:
This doesn't seem to be used anywhere.

Reply:
Good catch. Thank you. I was initially implementing a lazy evaluation of the half transform to work with the `full` method. Removed this.

Reply:
Got it.